src/root-redirect.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="refresh" content="1;url=/#LATEST_VERSION#/docs/">
```

Comment:
We have set basics.html in S3 AS root document
On 11 Mar 2016 13:14, "Konrad Dzwinel" <notifications@github.com> wrote:

> In src/root-redirect.html
> <https://github.com/brainly/style-guide/pull/680#discussion_r55822599>:
>
> > @@ -0,0 +1,14 @@
> > +<!DOCTYPE HTML>
> > +<html lang="en-US">
> > +<head>
> > +  <meta charset="UTF-8">
> > +  <meta http-equiv="refresh" content="1;url=/#LATEST_VERSION#/docs/">
>
> @aju <https://github.com/aju> maybe you thought about this one? That I
> should redirect to /version/docs/basics.html instead of /version/docs?
> /version/docs, for some reason, opens basics.html file by default
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/brainly/style-guide/pull/680/files#r55822599>.
>